exchanges/kucoin/kucoin.go

```diff
@@ -682,18 +644,17 @@ func (ku *Kucoin) InitiateIsolatedMarginBorrowing(ctx context.Context, symbol, c
 	if ccy == "" {
 		return nil, currency.ErrCurrencyCodeEmpty
 	}
-	params := make(map[string]interface{})
-	params["symbol"] = symbol
-	params["currency"] = ccy
 	if borrowStrategy == "" {
 		return nil, errors.New("borrowStrategy can not be empty")
 	}
-	params["borrowStrategy"] = borrowStrategy
 	if size == 0 {
 		return nil, errors.New("size can not be zero")
 	}
+	params := make(map[string]interface{})
```

Comment:
This endpoint is deprecated:
https://www.kucoin.com/announcement/the-kucoin-api-upgrade-has-been-completed-2023-09
> 19. Deprecated: Place Isolated Borrow Order, abandon the "POST /api/v1/isolated/borrow" interface and replace it with the "POST /api/v3/margin/borrow" interface

I think there are more updates for Kucoin than in this PR